docs/Makefile

```diff
@@ -40,30 +40,14 @@ spark-apidoc:
   mvn --no-transfer-progress clean install -DskipTests -Dspotless.check.skip=true && \
 	mvn --no-transfer-progress scala:doc
 
-.PHONY: html
-html: cpp-apidoc spark-apidoc
-	$(SPHINXBUILD) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
-	rm -fr $(BUILDDIR)/html/spark/reference
-	cp -fr $(ROOTDIR)/spark/graphar/target/site/scaladocs $(BUILDDIR)/html/spark/reference/
-	cd $(ROOTDIR)/java && \
-	mvn --no-transfer-progress -P javadoc javadoc:aggregate \
-				-Dmaven.antrun.skip=true \
-				-DskipTests \
-				-Djavadoc.output.directory=$(ROOTDIR)/docs/$(BUILDDIR)/html/java/ \
-				-Djavadoc.output.destDir=reference \
-				--quiet
-	@echo
-	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
-
 .PHONY: pyspark-apidoc
 pyspark-apidoc:
-	cd $(ROOTDIR)/pyspark && \
-	poetry run sphinx-apidoc -o $(ROOTDIR)/docs/pyspark/api graphar_pyspark/
+	cd ../pyspark && poetry run sphinx-apidoc -o ../docs/pyspark/api graphar_pyspark/
 
 .PHONY: html-poetry
-html-poetry:
-	cd $(ROOTDIR)/pyspark && \
-	poetry run bash -c "cd $(ROOTDIR)/docs && $(SPHINXBUILD) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html"
+html-poetry: cpp-apidoc spark-apidoc pyspark-apidoc
```

Comment:
I suggest use the `html` to keep consistent.

Reply:
Done